test/zippy/mzcompose.py

```diff
@@ -179,6 +184,8 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
 
         c.up("testdrive", persistent=True)
 
+        setup_default_ssh_test_connection(c)
```

Comment:
See line 183, materialized is removed there, so we can't set up the ssh connection here.

Reply:
> See line 183, materialized is removed there, so we can't set up the ssh connection here.

Thank you @def-, this (plus upping `ssh-bastion-host`) was the fix.